functions.php

```diff
@@ -1,13 +1,23 @@
-<?php 
+<?php
+/**
+ * Theme functions and definitions
+ *
+ * @package HelloElementor
+ */
 
+/**
+ * Load child theme css and optional scripts
+ *
+ * @return void
+ */
 function hello_elementor_child_enqueue_scripts() {
 	wp_enqueue_style(
 		'hello-elementor-child',
 		get_stylesheet_directory_uri() . '/style.css',
 		[
-			'hello-elementor'
+			'hello-elementor',
 		],
 		'1.0.0'
 	);
 }
-add_action( 'wp_enqueue_scripts', 'hello_elementor_child_enqueue_scripts' );
+add_action( 'wp_enqueue_scripts', 'hello_elementor_child_enqueue_scripts', 99 );
```

Comment:
```suggestion
add_action( 'wp_enqueue_scripts', 'hello_elementor_child_enqueue_scripts' );
```

Reply:
That goes against the solution, the priority is to fix #3

Reply:
This will fix this: https://github.com/elementor/hello-theme-child/pull/5/commits/25a048901d16199a86a2fa71b4470a2cd85c833a

Reply:
This will fix that issue: https://github.com/elementor/hello-theme-child/pull/5#discussion_r316047031

Reply:
@pless84,
Once you update the dependency array, there will be no need to fiddle with the priority.